user_management/src/user_management/settings.py

```diff
@@ -33,7 +33,7 @@
 PASSWORD_MAX_LENGTH = 30
 ELO_DEFAULT = 500
 TLD_MAX_LENGTH = 15
-FORGOT_PASSWORD_CODE_MAX_LENGTH = 12
+FORGOT_PASSWORD_CODE_MAX_LENGTH = 6
```

Comment:
Rename to `FORGOT_PASSWORD_CODE_LENGTH` as max length implies it could be shorter